src/main/java/aquality/appium/mobile/configuration/ApplicationProfile.java

```diff
@@ -42,4 +42,9 @@ public URL getRemoteConnectionUrl() {
             throw new InvalidArgumentException(String.format("Address %1$s provided in configuration by key %2$s is not valid", address, key));
         }
     }
+
+    @Override
+    public String getPackageNameWithScreens() {
```

Comment:
maybe getScreensPackageName sounds better?

Reply:
Not sure. This method is just a getter of the property from JSON. It should be the same as property.